client/allocrunner/taskrunner/task_runner_getters.go

```diff
@@ -61,11 +61,15 @@ func (tr *TaskRunner) getDriverHandle() *DriverHandle {
 	return tr.handle
 }
 
-// setDriverHanlde sets the driver handle and creates a new result proxy.
+// setDriverHanlde sets the driver handle, creates a new result proxy, and
```

Comment:
Hanlde typo

Reply:
Result proxy?

Reply:
Good catches. Removed a couple old result proxy comments (it existed briefly before the new plugin's waitch existed and offered the same functionality).